website/docs/r/bot_channels_registration.html.markdown

```diff
@@ -65,6 +65,8 @@ The following arguments are supported:
 
 * `icon_url` - (Optional) The icon URL to visually identify the Bot Channels Registration.
 
+-> **NOTE:** In v4.0 of the provider the `icon_url` will have a default value of `https://docs.botframework.com/static/devportal/client/images/bot-framework-default.png`.
```

Comment:
Let's omit the 4.0 note in the docs, we will add a full list of properties that have been affect in the upgrade guide closer to the release 
```suggestion
```

Reply:
Removed it.